docs/index.md

```diff
@@ -23,13 +23,41 @@
 
 Ralph is a toolbox for your learning analytics, it can be used as a:
 
-- **[LRS](https://en.wikipedia.org/wiki/Learning_Record_Store)**, a HTTP API server to collect xAPI statements (learning events), following the [ADL LRS standard](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Communication.md#partthree)
-- **command-line interface** (CLI), to build data pipelines the UNIX-way™️,
-- **library**, to fetch learning events from various backends, (de)serialize or
+- **[LRS](./tutorials/lrs.md)**, an HTTP API server to collect xAPI statements (learning events), following the [ADL LRS standard](https://github.com/adlnet/xAPI-Spec/blob/master/xAPI-Communication.md#partthree)
+- **[command-line interface](./tutorials/cli.md)** (CLI), to build data pipelines the UNIX-way™️,
+- **[library](./tutorials/library.md)**, to fetch learning events from various backends, (de)serialize or
     convert them from and to various standard formats such as
     [xAPI](https://adlnet.gov/projects/xapi/), or
     [openedx](https://docs.openedx.org/en/latest/developers/references/internal_data_formats/tracking_logs/index.html)
 
+## What is an LRS?
```

Comment:
You can add an info admonition at the end of this subsection to inform readers that the new documentation is the following one: https://opensource.ieee.org/xapi. However, Ralph still supports last 1.X version of xAPI

Reply:
Great idea, thanks!